demo/demo-templates/base.html

```diff
@@ -0,0 +1,9 @@
+{% extends "default:base.html" %}
+
+{% block content %}
+{{ super() }}
+<footer>
+  <h1>Unused: theoretically this could be a custom footer</h1>
```

Comment:
This didn't work, but I'd imagine a future extension to the Datasette API might look something like this.